src/environments/environment.ts

```diff
@@ -15,7 +15,6 @@ export const environment = {
   clarkReportsUrl: 'http://localhost:9001',
   s3Bucket: 'clark-dev-file-uploads',
   s3BucketRegion: 'us-east-1',
-  s3Endpoint: 'http://localhost:9000',
```

Comment:
why is this removed?

Reply:
Hardcoded it so we don't have failing builds because other environment files have missing attribute.